projects/core/src/lib/util/components/backend-status-page/backend-status-page.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="container" fxLayout="row" fxLayoutAlign="center center">
+    <mat-card class="info" *ngIf="backendInfo">
+        <mat-card-title>
+            <mat-icon mat-card-md-image svgIcon="geoengine:logo-alt" aria-label="Geo Engine" [inline]="true"></mat-icon>
+            <mat-card-title>Backend Status</mat-card-title>
+            <mat-card-subtitle>Details about the connected backend</mat-card-subtitle>
+        </mat-card-title>
+        <mat-card-content>
+            <mat-list-item>
+                <h3 matLine>backend version</h3>
+                <p matLine>{{ backendInfo.version }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>backend features</h3>
+                <p matLine>{{ backendInfo.features }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>build date</h3>
+                <p matLine>{{ backendInfo.buildDate }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>commmit hash</h3>
+                <p matLine>{{ backendInfo.commitHash }}</p>
+            </mat-list-item>
+        </mat-card-content>
+
+        <mat-card-actions>
+            <button mat-button color="primary" (click)="goBack()">go back</button>
+            <button mat-button (click)="refresh()">refresh status</button>
+        </mat-card-actions>
+    </mat-card>
+
+    <mat-card *ngIf="error">
+        <mat-card-title>
+            <mat-icon mat-card-md-image svgIcon="geoengine:logo-alt" aria-label="Geo Engine" [inline]="true"></mat-icon>
+            <mat-card-title>Backend Error</mat-card-title>
+            <mat-card-subtitle>The backend is not available!</mat-card-subtitle>
+        </mat-card-title>
+
+        <mat-card-content>
+            <mat-list-item>
+                <h3 matLine>Error status code</h3>
+                <p class="error" matLine>{{ error.status }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error status</h3>
+                <p class="error" matLine>{{ error.statusText }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error name</h3>
+                <p class="error" matLine>{{ error.name }}</p>
+            </mat-list-item>
+            <mat-list-item>
+                <h3 matLine>Error message</h3>
+                <p class="error" matLine>{{ error.message }}</p>
+            </mat-list-item>
+        </mat-card-content>
+
+        <mat-card-actions>
+            <button mat-button color="primary" (click)="goBack()">reload UI</button>
```

Comment:
1. I go to [the map](http://localhost:4200/#/map)
2. I start the backend
3. I hit this reload UI button
4. I'm suddenly on http://localhost:4200/#/signin

I would expect to go to the map

Reply:
please check again. i changed how that works

Reply:
Still the same for me

Reply:
yes thats what i do. I changed the logic so please try again.
[8ce08b4](https://github.com/geo-engine/geoengine-ui/pull/512/commits/8ce08b4826def6335875fb791d890546b1617e5a)

Reply:
[go to map fail.webm](https://user-images.githubusercontent.com/4947061/202188340-9414c2c5-eab6-4014-8ec6-d064f008a3ab.webm)

Reply:
i tried a different approach this time and also disabled going back to the map while the state is still an error.
[c0819b5](https://github.com/geo-engine/geoengine-ui/pull/512/commits/c0819b5069984039b20301f898670464cc075e01)